doc/source/whatsnew/v0.21.0.txt

```diff
@@ -24,6 +24,7 @@ New features
   <https://www.python.org/dev/peps/pep-0519/>`_ on most readers and writers (:issue:`13823`)
 - Added `__fspath__` method to :class`:pandas.HDFStore`, :class:`pandas.ExcelFile`,
   and :class:`pandas.ExcelWriter` to work properly with the file system path protocol (:issue:`13823`)
+- Added `limit_area` parameter to `DataFrame.interpolate()` method allowing further control of which NaNs are replaced (:issue:`16284`)
```

Comment:
show a small sub-section example here of why this parameter is useful (take the examples from the docs you wrote above). and provide a pointer to the docs (again which you wrote)